Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/orientation.h

```diff
@@ -97,25 +101,55 @@ bool is_outward_oriented(const PolygonMesh& pmesh,
   VPMap vpmap = choose_param(get_param(np, vertex_point),
                              get_const_property_map(vertex_point, pmesh));
   //Kernel
-  typedef typename GetGeomTraits<PolygonMesh, NamedParameters>::type Kernel;
-
-  internal::Compare_vertex_points_xyz_3<typename Kernel::Less_xyz_3, VPMap >
-    less_xyz(vpmap);
+  typedef typename GetGeomTraits<PolygonMesh, NamedParameters>::type GT;
+  GT gt = choose_param(get_param(np, geom_traits), GT());
```

Comment:
just a remark: it seems that the API of PMP does not allow to get the geom traits by a reference. So the geom traits objects used by PMP have to be lightweight.